Description: Remove the wasm32 architecture from Cargo.toml
Author: Nadzeya Hutsko <nadzya.info@gmail.com>
Forwarded: not-needed
Last-Update: 2025-08-22
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -50,72 +50,3 @@
 [features]
 debug_automatic_glGetError = []
 debug_trace_calls = []
-
-[target.'cfg(target_arch = "wasm32")'.dependencies.js-sys]
-version = "~0.3"
-
-[target.'cfg(target_arch = "wasm32")'.dependencies.slotmap]
-version = "1"
-
-[target.'cfg(target_arch = "wasm32")'.dependencies.wasm-bindgen]
-version = "~0.2"
-
-[target.'cfg(target_arch = "wasm32")'.dependencies.web_sys]
-version = "~0.3.60"
-features = [
-    "Document",
-    "Element",
-    "HtmlCanvasElement",
-    "HtmlImageElement",
-    "HtmlVideoElement",
-    "ImageBitmap",
-    "ImageData",
-    "VideoFrame",
-    "WebGlActiveInfo",
-    "WebGlBuffer",
-    "WebGlFramebuffer",
-    "WebGlProgram",
-    "WebGlQuery",
-    "WebGlRenderbuffer",
-    "WebGlRenderingContext",
-    "WebGl2RenderingContext",
-    "WebGlSampler",
-    "WebGlShader",
-    "WebGlShaderPrecisionFormat",
-    "WebGlSync",
-    "WebGlTexture",
-    "WebGlTransformFeedback",
-    "WebGlUniformLocation",
-    "WebGlVertexArrayObject",
-    "Window",
-    "AngleInstancedArrays",
-    "ExtBlendMinmax",
-    "ExtColorBufferFloat",
-    "ExtColorBufferHalfFloat",
-    "ExtDisjointTimerQuery",
-    "ExtFragDepth",
-    "ExtShaderTextureLod",
-    "ExtSRgb",
-    "ExtTextureFilterAnisotropic",
-    "OesElementIndexUint",
-    "OesStandardDerivatives",
-    "OesTextureFloat",
-    "OesTextureFloatLinear",
-    "OesTextureHalfFloat",
-    "OesTextureHalfFloatLinear",
-    "OesVertexArrayObject",
-    "WebglColorBufferFloat",
-    "WebglCompressedTextureAstc",
-    "WebglCompressedTextureEtc",
-    "WebglCompressedTextureEtc1",
-    "WebglCompressedTexturePvrtc",
-    "WebglCompressedTextureS3tc",
-    "WebglCompressedTextureS3tcSrgb",
-    "WebglDebugRendererInfo",
-    "WebglDebugShaders",
-    "WebglDepthTexture",
-    "WebglDrawBuffers",
-    "WebglLoseContext",
-    "OvrMultiview2",
-]
-package = "web-sys"
